core/test/client/unit/models/role_test.js

```diff
@@ -0,0 +1,22 @@
+import {
+    describeModel,
+    it
+} from 'ember-mocha';
+
+describeModel('role', function () {
+    it('provides a lowercase version of the name', function () {
+        var model = this.subject({
+            name: 'Author'
+        });
+
+        expect(model.get('name')).to.equal('Author');
+        expect(model.get('lowerCaseName')).to.equal('author');
+
+        Ember.run(function () {
```

Comment:
Cool, thanks =)

On Sun, Feb 1, 2015 at 7:30 PM, Jason Williams <notifications@github.com>
wrote:

> In core/test/client/unit/models/role_test.js
> <https://github.com/TryGhost/Ghost/pull/4880#discussion_r23904807>:
>
> > @@ -0,0 +1,22 @@
> > +import {
> > +    describeModel,
> > +    it
> > +} from 'ember-mocha';
> > +
> > +describeModel('role', function () {
> > +    it('provides a lowercase version of the name', function () {
> > +        var model = this.subject({
> > +            name: 'Author'
> > +        });
> > +
> > +        expect(model.get('name')).to.equal('Author');
> > +        expect(model.get('lowerCaseName')).to.equal('author');
> > +
> > +        Ember.run(function () {
>
> Anything that has an async side effect (like a binding) has to be run in
> an Ember.run() when Ember is set to testing mode.
>
> So, since role.name is bound to lowerCaseName by way of a computed
> property, it has to go inside Ember.run().
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TryGhost/Ghost/pull/4880/files#r23904807>.
>